core/src/test/java/org/infinispan/jmx/MvccLockManagerMBeanTest.java

```diff
@@ -79,7 +83,12 @@ public void testNumberOfLocksAvailable() throws Exception {
 
       assertAttributeValue("NumberOfLocksHeld", 1);
       assertAttributeValue("NumberOfLocksAvailable", initialAvailable - 1);
-      tm.getTransaction().runCommit(true);
+      try {
+         tm.getTransaction().runCommit(true);
+         fail("RollbackException expected!");
```

Comment:
I convinced myself with following argument: without the exception, you (== or anyone who wrote some test) are not sure if the transaction rolledback because it was forced or if you have some error in the test. 
If you don't wanna write the ```try-catch``` I can do one of the following:
* create an helper method that checks if everything runs as expected
* don't throw the exception if ```forceRollback=true``` and the transaction rolledback because of it.